src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs

```diff
@@ -643,7 +643,7 @@ public async Task<IStorageHistory> RestoreItemsFromTrashAsync(IList<IStorageItem
             using var r = cancellationToken.Register(CancelOperation, operationID, false);
 
             var moveResult = new ShellOperationResult();
-            var (status, response) = await FileOperationsHelpers.MoveItemAsync(source.Select(s => s.Path).ToArray(), destination.ToArray(), false, NativeWinApiHelper.CoreWindowHandle.ToInt64(), operationID);
+            var (status, response) = await FileOperationsHelpers.MoveItemAsync(source.Select(s => s.Path).ToArray(), destination.ToArray(), false, NativeWinApiHelper.CoreWindowHandle.ToInt64(), operationID, progress);
```

Comment:
A little beyond the scope of this PR but is there any way to consolidate some of this code?